doc/manual/source/man/dnst-signzone.rst

```diff
@@ -0,0 +1,108 @@
+dnst signzone
+===============
+
+Synopsis
+--------
+
+:program:`dnst signzone` ``[OPTIONS]`` ``<ZONEFILE>`` ``<KEY>...``
+
+Description
+-----------
+
+**dnst signzone** signs the zonefile with the given key(s).
+
+Keys must be specified by their base name (usually ``K<name>+<alg>+<id>``),
+i.e. WITHOUT the ``.private`` or ``.key`` extension. Both ``.private`` and
+``.key`` files are required.
+
+A date can be a timestamp (seconds since the epoch), or of the form
+``<YYYYMMdd[hhmmss]>``.
+
+
+Options
+-------
+
+.. option:: <ZONEFILE>
+
+      The zonefile to sign.
+
+.. option:: <KEY>...
+
+      The keys to sign the zonefile with.
+
+.. option:: -b
+
+      Add comments on DNSSEC records.
+
+.. option:: -d
+
+      Do not add used keys to the resulting zonefile.
+
+.. option:: -e <DATE>
+
+      Set the expiration date of signatures to this date. Defaults to
+      4 weeks from now.
+
+.. option:: -f <FILE>
+
+      Write signed zone to file. Use ``-f -`` to output to stdout. Defaults to
+      ``<ZONEFILE>.signed``.
+
+.. option:: -i <DATE>
+
+      Set the inception date of signatures to this date. Defaults to now.
+
+.. option:: -o <domain>
+
+      Set the origin for the zone (for zonefiles with relative names and no
+      $ORIGIN).
+
+.. option:: -u
+
+      Set SOA serial to the number of seconds since Jan 1st 1970.
+
+      If this would NOT result in the SOA serial increasing it will be
+      incremented instead.
+
+.. option:: -n
+
+      Use NSEC3 instead of NSEC. If specified, you can use extra options (see
+      :ref:`dnst-signzone-nsec3-options`).
+
+.. option:: -H
+
+      Hash only, don't sign.
+
+.. option:: -h, --help
+
+      Print the help text (short summary with ``-h``, long help with
+      ``--help``).
+
+
+.. _dnst-signzone-nsec3-options:
+
+NSEC3 options
+--------------------------------
+
+NSEC3 options for use with ``-n``.
+
+.. option:: -a <ALGORITHM>
```

Comment:
```suggestion
.. option:: -a <ALGORITHM NUMBER OR MNEMONIC>
```

Reply:
I wonder if we shouldn't remove this argument entirely from `dnst` as the only supported algorithm is SHA-1.

Reply:
Yeah, might be a good idea, as it serves no real purpose currently.